flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/catalog/CatalogManager.java

```diff
@@ -455,6 +457,94 @@ public void createTable(CatalogBaseTable table, ObjectIdentifier objectIdentifie
 			"CreateTable");
 	}
 
+	/**
+	 * Creates a database in a given fully qualified path.
+	 * @param catalogName
+	 * @param databaseName
+	 * @param database
+	 * @param ignoreIfExists If false exception will be thrown if a database exists in the given path.
+	 */
+	public void createDatabase(String catalogName,
+					String databaseName,
+					CatalogDatabase database,
+					boolean ignoreIfExists,
+					boolean ignoreNoCatalog) {
```

Comment:
Why do we need "ignoreNoCatalog"? I see callers are just passing "false" to this. I think we can just assume it's false all  the time.

Reply:
It provides the ability to ignore the exception if there is no corresponding catalog like the other catalog method behavior. I think we can keep this and so that it's more convenient to use later.